src/test/java/com/raisetech/inventoryapi/ProductServiceImplTest.java

```diff
@@ -57,4 +59,24 @@ class ProductServiceImplTest {
         verify(productMapper, times(1)).createProduct(product);
 
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "100"})
+    public void 商品IDを指定して更新したときに商品情報が更新されること(int id) throws Exception {
+        String initialName = "Bolt";
+        String renewedName = "Shaft";
+        when(productMapper.findById(id)).thenReturn(Optional.of(new Product(id, initialName)));
+        productServiceImpl.updateProductById(id, renewedName);
+        verify(productMapper, times(1)).updateProductById(id, renewedName);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0})
+    public void 存在しない商品IDを指定して更新したときに期待通り例外を返すこと(Integer id) throws Exception {
```

Comment:
findByIdの引数の方はintなので、intにしておいた方が良さそうですね！
```suggestion
    public void 存在しない商品IDを指定して更新したときに期待通り例外を返すこと(int id) throws Exception {
```

Reply:
77bcedc916a6ca0fc9dd5d1db1202b34704f45b7

修正しました！